lobster/tools/codebeamer/codebeamer.py

```diff
@@ -152,23 +152,30 @@ def get_many_items(cb_config, item_ids):
     return rv
 
 
-def get_query(mh, cb_config, query_id):
+def get_query(mh, cb_config, query):
     assert isinstance(mh, Message_Handler)
     assert isinstance(cb_config, dict)
-    assert isinstance(query_id, int)
+    assert isinstance(query, (int, str))
     rv = []
+    url = ""
     page_id = 1
     total_items = None
 
     while total_items is None or len(rv) < total_items:
         print("Fetching page %u of query..." % page_id)
-        url = "%s/reports/%u/items?page=%u&pageSize=%u" % \
-            (cb_config["base"],
-             query_id,
-             page_id,
-             cb_config["page_size"])
-        data = query_cb_single(cb_config, url)
-        assert len(data) == 4
+        if query is not None:
```

Comment:
This check is unnecessary as on line 158 an assert statement is added to check if query is `int` or `str`.